cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -24,7 +24,7 @@
 {%- macro _classes( nav_depth, suffix='', additional_suffix='' ) -%}
 {%- set general_class = base_class ~ '_content' -%}
 {%- set depth_class = general_class ~ '-' ~ nav_depth -%}
-{%- set additional_class = depth_class ~ '__' ~  additional_suffix if additional_suffix else '' -%}
+{%- set additional_class = depth_class ~ suffix ~ '__' ~  additional_suffix if additional_suffix else '' -%}
```

Comment:
If there are no additional suffixes, will the class end with `__`?

Reply:
I think additional_class will just be set to '' if there's no additional_suffix value.

Reply:
Ahhh, thanks, I was thinking the condition was only for the last item in the combined string.